deno/send_message/src/functions/send_message_discord_webhook.ts

```diff
@@ -0,0 +1,23 @@
+export default async function (env: any, message: string) {
+  const webhook = env["DISCORD_WEBHOOK_URL"];
+  console.log(webhook);
+
+  const res = await fetch(webhook, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      content: message,
+    }),
+  });
+  if (res.status !== 204) {
+    console.log(res);
+    return {
+      success: false,
+      error: "Failed to send message, check the webhook URL",
```

Comment:
Would it be more helpful to provide the error response from discord?

Reply:
Discord webhook doesn't send any errors

It is sends some weird response

Reply:
What response did you get? I got this for an invalid webhook:

```shell
$ curl -X POST -H 'Content-Type: application/json' -H 'Accept: application/json' https://discord.com/api/webhooks/123/123 -d ''
{"message": "Unknown Webhook", "code": 10015}
```

The message and [error code](https://discord.com/developers/docs/topics/opcodes-and-status-codes) might be helpful.

Reply:
ok let me check